src/map/items/item_weapon.cpp

```diff
@@ -47,13 +47,20 @@ CItemWeapon::CItemWeapon(uint16 id)
     m_delay          = 8000;
     m_baseDelay      = 8000; // this should only be needed for mobs (specifically mnks)
     m_maxHit         = 0;
+    m_DPS            = 0.0;
     m_ranged         = false;
     m_twoHanded      = false;
     m_wsunlockpoints = 0;
 }
 
 CItemWeapon::~CItemWeapon() = default;
 
+/************************************************************************
+ *                                                                       *
+ *   reset weapon delay to base delay                                    *
+ *                                                                       *
+ ************************************************************************/
```

Comment:
I gotta say, I know we do this all over (because our predecessors did) but I have the giant brick style of commenting, and would rather we just use the line we need without the fancy boxes. `#tag for future dev discussion`